scripts/sanitise-db.sql

```diff
@@ -8,19 +8,24 @@ DECLARE
     query TEXT;
     count_query TEXT;
     pre_update_count INTEGER;
+    table_specific_condition TEXT := '';
 BEGIN
     RAISE NOTICE 'Anonymising table: %, data_blob key: %, with lorem ipsum.', UPPER(table_name), UPPER(data_blob_key);
 
+    IF table_name = 'project_progress' THEN
```

Comment:
I think we're good to remove the table-specific condition stuff now - Stuart from the Towns Fund team has replied to Sam in the TF Hotline Teams chat saying that he will get a revision one way or the other for the constraint-violating row in the `project_progress` table - we'll just hold off merging this until the database update has been applied